### ISSUE TITLE
Question on Telerik Namespace for components

### ISSUE THREAD

**Issue by BenHayat** [telerik/blazor-ui#20]
Looking at the demo codes (razor pages) I see some pages like Calendar.razor uses both Telerik.Blazor namespace and the fully defined namespace for the Calendar
@using Telerik.Blazor
@using Telerik.Blazor.Components.Calendar

And in most other samples I only see the fully defined namespace for that component and not the Telerik.Blazor.

This has got me confused, as my thinking was Telerik.Blazor is the core component that ALL components use & share and then each component, i.e. Calendar has extra features and that's why we need to include it as using statement. But I find some inconsistency across some of the pages I was looking at.

What namespace **_hierarchy_** are being used with Telerik framework?
Perhaps the answer should also be in the docs to make it easier what using statement we should be using.
If I have missed it, please let me know.

Thanks!
..Ben

**Comment by ivanchev** [comment]
The Telerik.Blazor namespace is added because we currently hold some of the enums in it. For instance the **IconName** static class.

This was with the intention of having them available everywhere without the need of including an extra namespace - for instance inside a grid page, you were able to automatically use IconName.

However after the namespace changes implemented in preview 4, this was turned on its head, and we're now forced to be extremely verbose, until they introduce the ability to define namespaces.

In other words - this will be changed and fixed, and I'll update this issue when we do.

**Comment by BenHayat** [comment]
Thanks for the explanation.

**Comment by marin-bratanov** [comment]
An improvement on the namespaces was implemented in the [2.0.0 release](https://docs.telerik.com/blazor-ui/upgrade/breaking-changes/2-0-0).

This repo will contain examples and is not going to be used as an issue tracker for the component suite itself.
